projects/plugins/crm/js/ZeroBSCRM.admin.invoicebuilder.js

```diff
@@ -1569,8 +1569,10 @@ function zbscrm_JS_calc_amount_due() {
 			// get
 			var v = jQuery( '.zbs-partial-value', ele ).text();
 
+			var label = jQuery( '.zlabel', ele ).text();
+
 			// detect +-
-			var multiplier = 1; // gets turned to -1 if negotive ()
+			var multiplier = 1; // gets turned to -1 if negative ()
 
 			// got -?
 			if ( v.includes( '(' ) ) {
```

Comment:
It's unclear to me when the value amount would include a bracket, so leaving this for any unknown cases.

Reply:
It's common in accounting for a negative value to be surrounded by parentheses instead of having the sign. I'm not sure if it's possible through normal flows to have them an invoice value (we block it client-side when manually adding an invoice, and our imports should have the proper sign), but apparently adding it in transactions then saving swaps things to a negative, so that's likely the intent.

Reply:
Thanks for the explanation!